ttnn/cpp/ttnn/tensor/types.hpp

```diff
@@ -71,6 +71,14 @@ inline bool is_floating_point(DataType dtype) {
     }
 }
 
+inline bool is_block_float(DataType dtype) {
```

Comment:
Please remove inline. Move implementation to cpp

Reply:
Done